### DISCUSSION TITLE
Line chart tooltip

### DISCUSSION THREAD

**Discussion by madhukesh048** [imaNNeo/fl_chart#1258] [Q&A] [answered]
HI, Tooltip in line chart disappears after you lift the touch. I need to keep that tooltip on even after you release the touch. Is it possible to do?

**Comment by imaNNeo** [comment, marked as answer]
Hi,

You can customize the touch functionality
Check the below sample code:

##### ScatterChartSample 2 ([Source Code](https://github.com/imaNNeo/fl_chart/blob/master/example/lib/presentation/samples/scatter/scatter_chart_sample2.dart))
<img src="https://github.com/imaNNeo/fl_chart/raw/master/repo_files/images/scatter_chart/scatter_chart_sample_2.gif" width="300" >

**Comment by FMorschel** [comment]
Just passing here to help anyone who was also having some trouble in making this work. (Even following the example, maybe this should be mentioned somewhere in the docs?) The `LineTouchData.handleBuiltInTouches` parameter **must** be set to `false`.

**Reply by imaNNeo** [reply]
Yes, you're right. Can you please add it somewhere in the docs?

**Reply by FMorschel** [reply]
Sorry, but I'm not sure where would be the best place for it. Probably on the `///` above the `handleBuiltInTouches` but somewhere else may be better or should appear in more than one place.

**Reply by imaNNeo** [reply]
I think it makes sense to add it on the `touchCallback`docs here:

https://github.com/imaNNeo/fl_chart/blob/24d0fc5065eb03c1813ac8b1824617f97e84a21e/lib/src/chart/base/base_chart/base_chart_data.dart#L98-L102

Also, it would be nice if you add it in the chart subclasses such as Line, Bar, Pie, Scatter and Radar chart data.
